app/models/user.rb

```diff
@@ -36,6 +41,8 @@ class User < ApplicationRecord
 
   delegate :full_path, to: :namespace
 
+  attribute :user_type, :integer, default: -> { User.user_types[:human] }
```

Comment:
**nitpick (blocking):** Can you move this line into the `has_user_type` model concern.  I believe it should work in the `included do` block.

Reply:
Updated in adfe4001c792f2935730fcb01ed3c9c93771ea9a